CHANGELOG.md

```diff
@@ -9,16 +9,31 @@ All notable changes to AET will be documented in this file.
 
 ## Unreleased
 **List of changes that are finished but not yet released in any final version.**
-- [PR-41](https://github.com/Cognifide/aet/pull/41) - [Click](https://github.com/Cognifide/aet/wiki/ClickModifier)
- and [Hide](https://github.com/Cognifide/aet/wiki/HideModifier) modifiers allows to use css selectors as parameters. 
- Added [Replace Text](https://github.com/Cognifide/aet/wiki/ReplaceTextModifier) modifier and it's tests.
-- [PR-39](https://github.com/Cognifide/aet/pull/39) - fixed filtering w3cHtml issues.
+- 
 
+## Version 2.0.2
```

Comment:
ok, so what you think about list below: 

- [PR-44](https://github.com/Cognifide/aet/pull/44) [ScreenCollector](https://github.com/Cognifide/aet/wiki/ScreenCollector) enhancement: added ability to take screenshots of parts of the page by using css or xpath element selectors
- [PR-41](https://github.com/Cognifide/aet/pull/41) Added new [Replace Text](https://github.com/Cognifide/aet/wiki/ReplaceTextModifier) modifier and  [Click](https://github.com/Cognifide/aet/wiki/ClickModifier)
 and [Hide](https://github.com/Cognifide/aet/wiki/HideModifier) Modifiers allows to use css selectors as parameters. 
- [PR-39](https://github.com/Cognifide/aet/pull/39) Fixed filtering w3cHtml issues and update nuvalidator to the newest version
- [PR-31](https://github.com/Cognifide/aet/pull/31) Fixed source report artifact error
- [PR-30](https://github.com/Cognifide/aet/pull/30) Fixed cookie report display in 'test' action mode




REMOVED:
- [PR-63](https://github.com/Cognifide/aet/pull/63) Update AET version in vagrantfile and updated changelog
- [PR-55](https://github.com/Cognifide/aet/pull/55) Introducing AET changelog.
- [PR-54](https://github.com/Cognifide/aet/pull/54) Upgrade for bobcat from 0.9.6 to 1.1.3
- [PR-53](https://github.com/Cognifide/aet/pull/53) Removing not needed parameter (legacy proxy)
- [PR-52](https://github.com/Cognifide/aet/pull/52) Update for documentation and default proxy
- [PR-51](https://github.com/Cognifide/aet/pull/51) Improvements for modifiers
- [PR-49](https://github.com/Cognifide/aet/pull/49) Fix for number of successful tests in feature file
- [PR-48](https://github.com/Cognifide/aet/pull/48) Upgrade for frontend plugin version to 1.3
- [PR-40](https://github.com/Cognifide/aet/pull/40) Fixes sonar issues
- [PR-33](https://github.com/Cognifide/aet/pull/33) Travis config updated for SonarQube
- [PR-29](https://github.com/Cognifide/aet/pull/29) Fix for ignored test